app/policies/user_policy.rb

```diff
@@ -1,27 +1,34 @@
 class UserPolicy < ApplicationPolicy
-  USER_MANAGEMENT_ROLES = %w(superadmin policy_and_abuse open_doors support tag_wrangling).freeze
-  POLICY_ROLES = %w(superadmin policy_and_abuse).freeze
+  # Roles that allow:
+  # - troubleshooting for a user
+  # - managing invitations
```

Comment:
"managing a user's invitations" or "managing invitations granted to a user," maybe? We're going to end up with a whole other policy class for managing who can send invitations to email addresses and such via the admin/invitations page, and I keep getting confused why `UserPolicy` affects invitations at all.